neuronsimulator/forms.py

```diff
@@ -14,53 +14,30 @@ class ParamForm(forms.Form):
 
     form_opt_field_dict = wfc2plot().get_form_opt_field_dict()
 
-    neuron_choices = [
-        (neu_id, neu_id)
-        for neu_id in Neuron.objects.all().values_list("name", flat=True)
-    ]
+    strains = wfc.strains
```

Comment:
I like getting the strains from the underlying library, but does the library also map these strains to atlas files? If not, perhaps we want to include that mapping here? There is no point is listing a strain for which we have no atlas (data).